met/src/libcode/vx_nc_util/nc_utils.h

```diff
@@ -216,8 +216,7 @@ extern char   get_char_val(NcVar *var, const int index);
 extern int    get_int_var(NcFile *, const char * var_name, const int index);
 extern int    get_int_var(NcVar *, const int index);
 
-extern double get_double_var(NcFile *, const char * var_name, const int index = 0);
```

Comment:
Howard, I noticed that this function disappears entirely and is not renamed.  Is that intended?

Reply:
Yes. There were two get_double_var(). There was no call of get_double_var(NcFile *). So I commented out and John deleted it.

Sorry the command is not correct: "level="(1,,)" ==> "level="(1,*,*)"